rct229/rulesets/ashrae9012019/section19/section19rule33.py

```diff
@@ -0,0 +1,76 @@
+            each_rule=Section19Rule33.HVACRule(),
+            index_rmr="baseline",
+            id="19-33",
+            description="Where no heating and/or cooling system is to be installed, and a heating or cooling system is being simulated only to meet the requirements described in Section G3.1-10 HVAC Systems proposed column c and d, "
+            "heating and/or cooling system fans shall simulated to be cycled ON and OFF to meet heating and cooling loads during occupied hours in the proposed design.",
+            ruleset_section_title="HVAC - General",
+            standard_section="Section G3.1-10 HVAC Systems proposed column c and d",
+            is_primary_rule=False,
+            rmr_context="ruleset_model_descriptions/0",
+            list_path="$.buildings[*].building_segments[*].heating_ventilating_air_conditioning_systems[*]",
+        )
+
+    def create_data(self, context, data):
+        rmd_u = context.user
+        rmd_p = context.proposed
+        applicable_HVAC_systems_cooling_list_p = (
+            get_proposed_hvac_modeled_with_virtual_cooling(rmd_u, rmd_p)
+        )
+        applicable_HVAC_systems_heating_list_p = (
+            get_proposed_hvac_modeled_with_virtual_heating(rmd_u, rmd_p)
+        )
+
+        applicable_HVAC_systems_list_p = (
+            applicable_HVAC_systems_cooling_list_p
+            + applicable_HVAC_systems_heating_list_p
+        )
+        return {"applicable_HVAC_systems_list_p": applicable_HVAC_systems_list_p}
+
+    class HVACRule(PartialRuleDefinition):
+        def __init__(self):
+            super(Section19Rule33.HVACRule, self).__init__(
+                rmrs_used=UserBaselineProposedVals(True, False, True),
+            )
+
+        def applicability_check(self, context, calc_vals, data):
+            hvac_p = context.proposed
+            hvac_id_p = hvac_p["id"]
+            applicable_HVAC_systems_list_p = data["applicable_HVAC_systems_list_p"]
+
+            return hvac_id_p in applicable_HVAC_systems_list_p
+
+        def get_manual_check_required_msg(self, context, calc_vals=None, data=None):
+            hvac_b = context.baseline
+            hvac_id_b = hvac_b["id"]
+
+            hvac_p = context.proposed
+            hvac_id_p = hvac_p["id"]
+
+            operation_during_occupied_p = getattr_(
+                hvac_p, "HVAC", "fan_system", "operation_during_occupied"
+            )
+
+            return (
+                f"It appears that {hvac_id_p} is only being simulated in the P_RMI to meet the requirements described in Section G3.1-10 HVAC Systems proposed column c and d for heating and/or cooling. "
+                f"Check that the heating and/or cooling system fans are simulated to be cycled ON and OFF to meet heating and/or cooling loads during occupied hours as applicable. "
+                f"Note that per the RMD the fan associated with {hvac_id_b} is operating as {operation_during_occupied_p} during occupied hours. "
```

Comment:
I think the description in the RDS is incorrect. This should also be `hvac_id_p`.

Reply:
Addressed